Documentation/Signoffs/LaunchingAndFiring.md

```diff
@@ -0,0 +1,93 @@
+
+## Analysis 
+
+**Constraint Solution 1:**
+The Launching and Firing subsystem shall use 16 AWG wire for the subsystems electrical connections. This wire has an ampacity rating of 10 amps over a distance of 12 inches. This wire size was selected by considering the largest ampacity draw that exists in the subsystem. For this the team decided to use the start/stall current (9.8 amps) of the 12V DC motor used in this subsystem to apply mechanical power to the firing mechanism. Using a table [4] that seems to simplify and directly quotes the NEC 310.15(B)(16), the team decided to use 14 AWG wire[5] than can handle over 15 Amps, leaving plenty of capacity in the wire, this wire will be sufficient to use for this application. 
+
+**Constraint Solution 2:**
+To allow the firing mechanism to operate without human interaction, a signal shall be generated from the Aiming subsystem when that subsystem has finished the aiming processes that will be used as a relay signal to operate the firing mechanism autonomously. The relay[3] will disconnect the 12-volt supply from the DC motor when the signal is not active, or low, and will connect the 12-volt supply to the DC motor when the signal is activated, or high, allowing the motor to operate. The relay will be connected to pin 1 of the Raspberry Pi 5 from the aiming subsystem and will pull 100mA at max (while operating) which is within the Raspberry Pi 5's ampacity limits for that pin. The relay, the Adafruit Power Relay FeatherWing G5LE-G is rated for 10 Amp resistive loads at 12 Volts DC. This is over the motor's stall ampacity and it also useful to consider that the team will be operating the motor at much ampacity than this at all times however this relay can handle 10 amps at 12 volts regardless. After the shot is and preloading sequence is completed, a proximity sensor (mounted by ME team) will sense a bolt during that part of the rotation cycle and will send a signal to the Raspberry Pi 5 GPIO 11 (Pin23) to disengage the PMDC motor allowing the motor to come to rest against the resistance of the cam in the ME team's design at a predetermined point each time.
```

Comment:
Is the Adafruit Power Relay 3191 using the G5LE-G or G5LE-14? Will the G5LE-14 carry 10A at 12VDC? Look carefully at the datasheets online.